docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: "3.8"
+
+services:
+  apiary:
+    image: robojackets/apiary
+    network_mode: host
+    environment:
+      APP_NAME: Apiary Local
+      APP_ENV: local
+      APP_KEY: base64:JVvxt2ojHHrHVIHhTeCGkjWan3dhQQZwTKnEuUhxF0g=
```

Comment:
Why is `APP_KEY` hardcoded?

Reply:
This is just for local use, so it doesn't really matter. But I did learn a way to generate one on the fly and set it as an environment variable, so I'm doing that instead.